README.md

```diff
@@ -0,0 +1,136 @@
+### Notebook Images
+
+These images were created to be used with Open Data Hub (ODH) or Red Hat Openshift Data Science (RHODS) with the Kubeflow Notebook Controller as the launcher.
+
+The notebooks support CPU and GPU running modes. GPU notebooks can be also spawned in clusters without GPU enabled using the CPU capabilities:
+
+**CPU Notebooks**
+- Minimal
+- Standard Data Science
+
+**GPU Notebooks**
+- CUDA
+- CUDA PyTorch
+- CUDA TensorFlow
```

Comment:
I wouldn't add this until we start implementing CPU/GPU notebooks. For now, we are only building the base image and we only want to add documentation about this image.